test/metabase/integrations/ldap_test.clj

```diff
@@ -64,7 +64,11 @@
 
     (testing "fail regular user login with bad password"
       (is (= false
-             (ldap/verify-password "cn=Sally Brown,ou=People,dc=metabase,dc=com" "password"))))))
+             (ldap/verify-password "cn=Sally Brown,ou=People,dc=metabase,dc=com" "password"))))
+
+    (testing "password containing dollar signs succeeds (#15145)"
+      (is (= true
+             (ldap/verify-password "cn=Fred Taylor,ou=People,dc=metabase,dc=com", "pa$$word"))))))
```

Comment:
You can actually do `(is (ldap/verify-password ...))` here instead of `(is (= true (ldap/verify-password ...)))`

Reply:
I did it this way to be consistent with the other test cases above it, which are all comparing explicitly against `true` or `false`